### PR TITLE
add ppc64le support

### LINKED ISSUE #97: Compiling VTK for POWER (ppc64le) PC
Issue: 
Hi! 

I would like to compile VTK with the Python wrapping for linux-ppc64le running RHEL 7.4. Specifically for an AC922, which contains an IBM POWER9 PC, and 4*V100 Nvidia GPUs. 

The go to method of building our Python packages is using conda, and therefore ideally I would like to make use of the feedstock, alongside conda-build. I noticed that linux-ppc64le is marked as disabled on the Readme, and so would like to understand the reasons for that.

I know that VTK-m has been [built successfully for the architecture](https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9458-vtk-m-lessons-from-building-a-visualization-toolkit-for-massively-threaded-architectures-v2.pdf), but I'm not sure that provides a Python wrapper. 

Could anyone provide any direction/pointers on how to compile for linux-ppc64le? 

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : vtk_python
    active env location : /home/tom/anaconda3/envs/vtk_python
            shell level : 2
       user config file : /home/tom/.condarc
 populated config files : /home/tom/.condarc
          conda version : 4.7.5
    conda-build version : not installed
         python version : 3.7.2.final.0
       virtual packages : __cuda=10.1
       base environment : /home/tom/anaconda3  (writable)
           channel URLs : https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/linux-ppc64le
                          https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/noarch
                          https://conda.anaconda.org/conda-forge/linux-ppc64le
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-ppc64le
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-ppc64le
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/tom/anaconda3/pkgs
                          /home/tom/.conda/pkgs
       envs directories : /home/tom/anaconda3/envs
                          /home/tom/.conda/envs
               platform : linux-ppc64le
             user-agent : conda/4.7.5 requests/2.22.0 CPython/3.7.2 Linux/3.10.0-957.10.1.el7.ppc64le rhel/7.6 glibc/2.17
                UID:GID : 1001:1001
             netrc file : None
           offline mode : False

```
</details>

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes #97.

cc: #154.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Fails on ppc64le with

`
/home/conda/feedstock_root/.scripts/build_steps.sh: line 75:   617 Killed                  conda $BUILD_CMD "${RECIPE_ROOT}" -m "${CI_SUPPORT}/${CONFIG}.yaml" --suppress-variables ${EXTRA_CB_OPTIONS:-} --clobber-file "${CI_SUPPORT}/clobber_${CONFIG}.yaml"
`

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -19,11 +19,11 @@ if [ -f "$PREFIX/lib/libOSMesa32${SHLIB_EXT}" ]; then
         "-DOSMESA_LIBRARY:FILEPATH=${PREFIX}/lib/libOSMesa32${SHLIB_EXT}"
     )
 
-    if [ "${OSNAME}" == Linux ]; then
+    if [[ "${travis_platform}" == linux-* ]]; then
```
Comment (matthiasdiener) [inline recipe/build.sh:22]: Aren't these running on Azure, not Travis?

Comment (isuruf) [reply recipe/build.sh]: should have been `target_platform` and not `travis_platform`

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/vtk, this is ready

**Comment by isuruf** [comment]
@conda-forge/vtk, ping

**Comment by isuruf** [comment]
@matthiasdiener, we can merge if you add yourself as a maintainer

**Comment by matthiasdiener** [comment]
> @matthiasdiener, we can merge if you add yourself as a maintainer

Ok, I added myself.

**Pull request merged**